### ISSUE TITLE
Volume*Price instead of just volume filtering

### ISSUE THREAD

**Issue by sword134** [tradytics/surpriver#3]
Hello, I really like what you have done here and I've been messing around with it a bit. However it's mind-boggling how you can only filter for volume and not volume*price. For example 100k volume in a 2$ stock over a 5 day period amounts to 200.000$ traded, however 10 volume in Berkshire Hathaway A amounts to 3.000.000$ traded over a day period. However with the current filter Berkshire Hathaway A will be filtered out and the penny stock will be kept.

**Comment by tradytics** [comment]
Oh so that's totally doable. In the volume filter, do you wanna take a quick pass to add the price as well. Otherwise, I'll do it.

**Comment by sword134** [comment]
@tradytics I am unfortunately not at my pc atm, will do when I get home if you haven't already

**Comment by Rajmehta123** [comment]
Agreed. This is an important parameter  in order to avoid filtering the high price stocks.
